### PR TITLE
Upgrade for 2.8: ContainerAware was deprecated in favour of ContainerAwareTrait

### PR DESCRIPTION
**Pull request by pamil**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: UPGRADE-2.8.md [hunk 1]
```diff
@@ -374,6 +374,9 @@ DependencyInjection
    </services>
    ```
 
+ * `Symfony\Component\DependencyInjection\ContainerAware` has been deprecated, use 
+    trait `Symfony\Component\DependencyInjection\ContainerAwareTrait` instead
```
Comment (stof) [inline UPGRADE-2.8.md]: I would say "use the trait or implement the interface manually", otherwise people needing to keep PPH 5.3 support will complain

Comment (pamil) [inline UPGRADE-2.8.md]: :+1:

File: UPGRADE-2.8.md [hunk 2]
```diff
@@ -374,6 +374,10 @@ DependencyInjection
    </services>
    ```
 
+ * `Symfony\Component\DependencyInjection\ContainerAware` has been deprecated, use 
+    use the `Symfony\Component\DependencyInjection\ContainerAwareTrait` trait or implement
```
Comment (xabbuh) [inline UPGRADE-2.8.md]: double "use"

Comment (xabbuh) [inline UPGRADE-2.8.md]: and there's no need for "trait" after the trait name

File: UPGRADE-2.8.md [hunk 3]
```diff
@@ -374,6 +374,10 @@ DependencyInjection
    </services>
    ```
 
+ * `Symfony\Component\DependencyInjection\ContainerAware` has been deprecated, use 
+    use the `Symfony\Component\DependencyInjection\ContainerAwareTrait` trait or implement 
+    the interface `Symfony\Component\DependencyInjection\ContainerAwareInterface` manually
```
Comment (xabbuh) [inline UPGRADE-2.8.md]: I would remove "interface" here too.

**Comment by fabpot** [comment]
Thank you @pamil.

**Pull request merged**